docs/ReleaseNotes.md

```diff
@@ -0,0 +1,77 @@
+# Antiplagiarism Release Notes
+This document provides information about fixes, enhancements, and key features that are available in Antiplagiarism.
+
+## Antiplagiarism 1.3
+Antiplagiarism 1.3 includes new features, enhancements, and bug fixes described in this section, as well as the features that have been implemented in the previous version.
+
+### Code Comparison Algorithm Enhancement
+Antiplagiarism now uses advanced code analysis algorithm from Roslyn which performs mich more complex comparison of syntax trees.
+This greatly improved the quality of analysis and allowed to find duplication even for small methods (about 5-10 lines).
+
+### Code Fragments in Data Grid
+The code fragments for the selected row can be shown directly in the data grid if you open the context menu with right click and select **Show Code**. 
+You can hide the code by right clicking on the selected row again and selecting **Hide Code** or by changing the selected row.
+
+### New Settings
+Antiplagiarism now has a new **Settings** panel. It can be expanded/collapsed by clicking on the button with a gear icon.
+
+#### Columns Visibility
+The **Column Visibility** button in the settings panel allows you to specify the columns visible in the data grid. 
+
+#### Show only rows exceeding threshold
+The **Show only rows exceeding threshold** button allows you to turn on mode in which all rows with **Similarity** value not exceeding **Threshold** are hidden.
+This feature can be combined with the filtering calculated results by the **Threshold** parameter.
```

Comment:
Not clear what this sentence means.

Reply:
When you run the analysis all the results will be exceeding the specified Threshold and marked with a corresponding sign.  But you can change Threshold value and the results will be updated 
This was described in the notes for the version 1.0.
When you turn on the "Show only rows exceeding threshold" mode only the results exceeding Threshold will be shown.  This is useful only if you change Threshold.
For example, you runned the analysis with Threshold 60% to obtain more results but now you want to see only very similar methods. So, you change the Threshold to 85% and turn on 
"Show only rows exceeding threshold". This way you don't need to re-run the analysis which can be a long operation